<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_KM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>K to M</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_GM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>G to M</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_BM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>B to M</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_KB">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>K  to B</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_BK">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>B to K</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_MB">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>M to B</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_GB">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>G to B</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_MK">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>M to K</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_GK">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>G to K</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLineEdit" name="line">
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
background-color: rgb(0, 0, 127);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>B: bite</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>M : megabite</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>K : kilobit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>G : gigabit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="btn_c">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 178);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>convert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_BG">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>B to G</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_KG">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>K to G</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_MG">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>M to G</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
